internal/pkg/manifest/job.go

```diff
@@ -36,7 +36,7 @@ type ScheduledJob struct {
 
 // ScheduledJobConfig holds the configuration for a scheduled job
 type ScheduledJobConfig struct {
-	Image          Image `yaml:",flow"`
+	ImageConfig    Image `yaml:",flow"`
```

Comment:
```suggestion
	ImageConfig    Image `yaml:"image,flow"`
```
Otherwise the unmarshaler will be looking for `imageConfig`. Will be fixed in #1473.